ci/no-std-check/Cargo.toml

```diff
@@ -9,7 +9,7 @@ ibc-types = { path = "../../crates/ibc-types", default-features = false, feature
   "serde",
   "mocks-no-std",
 ] }
-ibc-proto = { version = "0.42.0", default-features = false }
+ibc-proto = { version = "0.42.2", default-features = false }
```

Comment:
Why the change to `ibc-proto` throughout, too? Is that optimistic, or was it required by the `ics23` bump?

Reply:
ibc-types updated its version of ibc-proto so i brought it into alignment